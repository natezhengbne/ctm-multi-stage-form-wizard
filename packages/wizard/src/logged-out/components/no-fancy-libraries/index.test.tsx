import { mount, shallow } from "enzyme";
import React from "react";
import NoFancyLibrariesPage from ".";

describe("The entry point of no fancy libraries stepper renders", () => {
  it("renders correctly", () => {
    let wrapper = shallow(<NoFancyLibrariesPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
