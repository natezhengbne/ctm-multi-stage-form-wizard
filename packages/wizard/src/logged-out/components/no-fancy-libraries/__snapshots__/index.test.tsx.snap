// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The entry point of no fancy libraries stepper renders renders correctly 1`] = `
<Fragment>
  <Memo(NavBar) />
  <StepIndicator
    value={33.33333333333333}
  />
  <StepPersonal
    activeStep={1}
    onNext={[Function]}
    onPrev={[Function]}
    step={1}
  />
  <StepAddress
    activeStep={1}
    onNext={[Function]}
    onPrev={[Function]}
    step={2}
  />
  <StepSummary
    activeStep={1}
    onPrev={[Function]}
    step={3}
  />
</Fragment>
`;
